nav2_docking/opennav_docking/test/test_docking_server.py

```diff
@@ -155,8 +173,8 @@ def nav_execute_callback(
                                       NavigateToPose.Result, NavigateToPose.Feedback]
     ) -> NavigateToPose.Result:
         goal = goal_handle.request
-        self.x = goal.pose.pose.position.x - 0.05
-        self.y = goal.pose.pose.position.y + 0.05
+        self.x = goal.pose.pose.position.x - self.period
+        self.y = goal.pose.pose.position.y + self.period
```

Comment:
I'm don't think the 0.05 here is actually related to the period - I think it's just intended to be a bit of random noise, right?